### DISCUSSION TITLE
Superset Version's

### DISCUSSION THREAD

**Discussion by hugopvl** [apache/superset#18025] [Q&A / Help]
Hello Guys, I'm new here, and I'm really enjoying the experience, but I have some doubts, I did the installation through docker-compose and when I check the version is: 0.0.0dev, this is referring to the last RC in this case it would be the 1.4.0rc4?
Or is it the latest stable version?

I ask this because I see that there is an option to assign a role to a dashboard, but in the version I installed it doesn't exist.

Thanks for listening.

**Comment by hugopvl** [comment]
> Hello Guys, I'm new here, and I'm really enjoying the experience, but I have some doubts, I did the installation through docker-compose and when I check the version is: 0.0.0dev, this is referring to the last RC in this case it would be the 1.4.0rc4? Or is it the latest stable version?
> 
> I ask this because I see that there is an option to assign a role to a dashboard, but in the version I installed it doesn't exist.
> 
> Thanks for listening.

ok, I just realized that I can enable that option in the feature flags, superset_config.py, but what is the way to get to those files, since I'm using docker ? thanks

**Reply by cycy9999** [reply]
I found that post: seem to be normal

https://stackoverflow.com/questions/69600229/superset-compile-problems-whlile-running-docker-latest-dev-or-latest

**Comment by cycy9999** [comment]
@hugopvl  I ve the same problem - I installed Apache superset using docker.

**Comment by eschutho** [comment]
@hugopvl which flag are you trying to enable? There are instructions in the README in the docker folder around local overrides which is where you would add a feature flag. https://github.com/apache/superset/blob/master/docker/README.md

**Comment by 1c7** [comment]
Hi, the version in top right corner "Settings" menu are not accurate

<img width="532" alt="image" src="https://user-images.githubusercontent.com/1804755/154642562-88357e2c-012e-49ae-a767-ec38dab26bbd.png">

if you are using Docker, you probably building your own Docker Image base on the official one.
you can select version using tag:
https://hub.docker.com/layers/apache/superset/1.4.1/images/sha256-5e1e0f0cc5ba4a6a955219f5655c110d67b5220df09977374f06809792ef231a?context=explore

## Like this:  
in your `Dockerfile`:  
```
FROM apache/superset:1.4.1
```
